packages/build/src/core/flags.js

```diff
@@ -134,7 +134,12 @@ Default: false`,
   },
   debug: {
     boolean: true,
-    describe: 'Print debugging information',
+    describe: 'Print user-facing debugging information',
+    hidden: true,
+  },
+  verbose: {
+    boolean: true,
+    describe: 'Print internal debugging information',
     hidden: true,
   },
```

Comment:
Given the need for this ASAP I'm ok with the flag, but what do you think of having debug levels in the future? It would be more suited IMO vs these flags which can be confusing

Reply:
Yes, I was thinking this should be the next step as well!

Note than from a user standpoint, they use the `NETLIFY_BUILD_DEBUG` environment variable for both `debug` and `verbose`, so there's currently a single level from their perspective. The buildbot and CLI do not pass the `verbose` flag, so the only difference is for the tests, which sets `verbose` as `false` and `debug` as `true`.